res/css/views/dialogs/_DevtoolsDialog.pcss

```diff
@@ -79,7 +79,7 @@ limitations under the License.
 .mx_DevTools_textarea {
     font-size: $font-12px;
     max-width: 684px;
-    min-height: 250px;
+    min-height: 32em;
```

Comment:
Where does this magic number come from? Can you add a comment to explain?

Reply:
Most of the code for this window(Dev Tools Edit) is a same as the previous window(Dev Tools) which works for the Dev Tools window but not for this window.

Issue:
Here,the min-Height was set to 250px(without min-height edit window shrinks to Zero) which was lesser than the previous window's comperitive Height.

Solution:
I've had changed the height to 32 em which is working fine...Rn.

But I've found out another issue with the window so I am working on that too and I've mentioned ity in the pull request.

Reply:
I've mad some changes according to requirement.